vignettes/analysing-n-grams.Rmd

```diff
@@ -336,6 +343,7 @@ When analyzing bigrams, we might want to look at the relationships between
 common terms. For this we can leverage the power of
 [igraph](http://igraph.org/r/) and
 [ggraph](https://cran.r-project.org/web/packages/ggraph/index.html).
+(#6)
```

Comment:
didn't you use ggpraph above for the histogram plot already?

Reply:
nope, that's `ggplot2`.